pdp/templates/cascade.html

```diff
@@ -137,12 +138,11 @@ <h2 class="sr-only">General Profile</h2>
                                 type="submit">Go!</button>
                     </span>
                 </div><!-- /input-group -->
-            </form></div></div>
-            <div class="row"><div class="col-lg-6">
-                <div ng-show="profile.impersonationNetid" class="alert alert-danger" role="alert">
-                    Impersonating UWNetID "{{ profile.impersonationNetid }}"
-                </div>
-            </div></div>
+            </form>
+        </div>
+        <div ng-show="profile.impersonationNetid" class="alert alert-danger" role="alert">
```

Comment:
can you put this div `<div ng-show="profile.impersonationNetid"  ...` inside of the `<div ng-if="profile.isAdmin">` directly above it? i think the behavior is the same but the intent was that that block only be available to admins.